build.sbt

```diff
@@ -119,16 +119,6 @@ lazy val runtime = CrossProject(
   .jvmSettings(
     Test / fork := true
   )
-  .jsSettings(
-    scalaJSStage := FastOptStage,
-    // While not exactlu ideal, this is only used in the invoker to assign a
-    // unique id to ensure measurements have unique ids. It's never exposed to
-    // the user and doesn't touch anything sensitve, so we should have no
-    // issues here. Still, I don't like having this, so we should try to
-    // replace it.
-    libraryDependencies += ("org.scala-js" %%% "scalajs-fake-insecure-java-securerandom" % "1.0.0")
-      .cross(CrossVersion.for3Use2_13)
```

Comment:
Re-iterating my comment https://github.com/scoverage/scalac-scoverage-plugin/pull/459#discussion_r885241553 for visibility.
> unfortunately this is false, this _is_ exposed to the user. If it's a helpful analogy: it would be like if using scoverage on the JVM somehow replaced the JVM's default `SecureRandom` with a fake, insecure one. Since it's a Java API (and not a private internal one) user code is _very_ exposed to this.

Reply:
Absolutely agree. It's far too easy to reach for, and you haven't been the only one.